aequilibrae/project/about.py

```diff
@@ -0,0 +1,84 @@
+import sqlite3
+import string
+from warnings import warn
+from aequilibrae.project.project_creation import create_about_table
+
+
+class About:
+    """Provides an interface for querying and editing the **about** table of an AequilibraE project"""
+
+    def __init__(self, conn: sqlite3.Connection):
+        self.__characteristics = []
+        self.__conn = conn
+        if self.__has_about():
+            self.__load()
+
+    def create(self):
+        """Creates the 'about' table for project files that did not previously contain it"""
+
+        if not self.__has_about():
+            create_about_table(self.__conn)
+            self.__load()
+        else:
+            warn('About table already exists', Warning)
+
+    def add_info_field(self, info_field: str) -> None:
+        """Adds new information field to the model
+
+            Args:
+                *info_field* (:obj:`str`): Name of the desired information field to be added.  Has to be a valid
+                Python VARIABLE name (i.e. letter as first character, no spaces and no special characters)
+
+            ::
+
+                p = Project()
+                p.open('my/project/folder')
+                p.about.add_info_field('my_super_relevant_field')
+                p.about.my_super_relevant_field = 'super relevant information'
+                p.about.write_back()
+        """
+        passed = True
+        for x in info_field:
+            if x not in string.ascii_lowercase + '_':
```

Comment:
not a big deal here, but this will be constructing a new, merged, array each iteration, checking if x is in it and then throwing it away. Obviously this isn't high performance code so not crucial here, but just a habit to avoid.

Reply:
> Reading through the About class... have you considered just using an ORM type helper like sqlalchemy?
> I feel that most of that would be simply expressed as a "project has many comments"

I did think about it, but QGIS for Windows does not ship with any ORM. This compatibility criterium is a real bitch, but it is one I need to keep for now.

Reply:
> not a big deal here, but this will be constructing a new, merged, array each iteration, checking if x is in it and then throwing it away. Obviously this isn't high performance code so not crucial here, but just a habit to avoid.

Fair point.  It should look better now.